packages/terra-application/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+Changelog
```

Comment:
Are there changes to the files in `packages/terra-application` that are different than the ones in this PR?
https://github.com/cerner/terra-framework/pull/678

Trying to figure out if this files need reviewed in this current PR or if they are the same from Tyler's PR.

Reply:
They should be the same as Tyler's PR, need to check if there is a dependency update though.